Bottom: 2b54cdc8ebf887aa319cd8a2734bf8aa0b4a9e77
Top:    1bc42484d4e56795533e9f39020d589c095d3082
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-08 20:49:13 +0000

Refresh of play-with-render-to-texture-st

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 4c2daa3..eeee803 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1723,7 +1723,7 @@ GhidDrawLayerGroup (int group, const BoxType * screen)
       r_search (Layer->text_tree, screen, NULL, text_callback, Layer);
 
       /* Draw pins, vias and pads on this layer */
-      if (rv) {
+      if (!is_outline) {
         if (!global_view_2d) {
           if (PCB->PinOn &&
               (group == bottom_group || group == top_group))
